pkg/cmd/gc/pods/pods_test.go

```diff
@@ -0,0 +1,61 @@
+package pods
```

Comment:
`pods_test` for package name.

Reply:
I would not be able to test internal methods if I do so. What do you think?

Reply:
we should not be testing internal methods, only test exported functions (there are exceptions to the rules though). This will force us to think about the external API.

Reply:
Hmm... but then tests will become more functional tests (testing more facades than single units of code, less precise) than unit testing. But if it is ok for you then I can change.

Reply:
Ok, I will try to test just exported methods.